deployments/k8s/deployment.yaml

```diff
@@ -69,7 +69,12 @@ data:
       cron-job-time-interval: "0h0m10s"         # format: XhYmZs
       dbname: ./accuknox-obs.db
       system-observability: true
-      network-observability: true
+      network-observability: false
```

Comment:
why are we disabling network-observability?

Reply:
@Ankurk99 We do not have network observability. We will be using kubearmor to get network observability and generic network logs + observability as well.

Reply:
Ah! so have we removed the cilium network observability support?

Reply:
We are storing logs into network_logs DB and also retreiving it for observability. But not being used. 
Which will be removed in future.